src/app/calculator/furnaces/charge-material/energy-form/energy-form.component.ts

```diff
@@ -155,4 +160,27 @@ export class EnergyFormComponent implements OnInit {
       this.formWidth = this.formElement.nativeElement.clientWidth;
```

Comment:
This function is never called so formWidth isn't set

Should be called afterViewInit and on window resize I think

`if(this.formElement)`